kimhaejung/controllers/postController.js

```diff
@@ -0,0 +1,70 @@
+const postService = require("../services/postService");
+
+const posting = async (req, res) => {
+  try {
+    const { title, content, userId, imageUrl } = req.body;
```

Comment:
like 참고해서 userId는 수정해 보세요.
```suggestion
    const { title, content, imageUrl } = req.body;
```